net-dns/maradns/maradns-3.3.03.ebuild

```diff
@@ -0,0 +1,110 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 systemd toolchain-funcs
+
+DEADWOOD_VER="3.3.03"
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="http://www.maradns.org/"
+SRC_URI="http://www.maradns.org/download/${PV%.*}/${PV}/${P}.tar.bz2"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="authonly examples ipv6 systemd"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	acct-group/maradns
+	acct-user/maradns
+	acct-user/duende
+	dev-lang/perl"
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# Use duende-ng.c.
+	cp tools/duende{-ng,}.c \
+		|| die "failed to rename duende-ng.c"
+	# replace "make" by "$(MAKE)" to avoid GNU Make warning
+	sed -i 's/\ make\ /\ \$\(MAKE\)\ /g' build/Makefile.linux \
+		|| die "can't patch build/Makefile.linux"
+
+	python_fix_shebang tools/bind2csv2.py
+}
+
+src_configure() {
+	tc-export CC
+	./configure $(use ipv6 && echo "--ipv6") || die "Failed to configure ${PN}"
```

Comment:
Use ```econf $(usex ipv6 '--ipv6' '')```` and ```die``` can be dropped.

Reply:
no, developer's configure script stopped with _Where [options] currently only supports "--ipv6"._